runelite-client/src/main/java/net/runelite/client/plugins/chatnotifications/ChatNotificationsConfig.java

```diff
@@ -44,7 +44,7 @@
 		position = 1,
 		keyName = "highlightWordsString",
 		name = "Highlight words",
-		description = "Highlights the following words in chat",
+		description = "Highlights the following comma-separated words in chat",
```

Comment:
This is more in line with existing config description wording, such as Chat Filter
```suggestion
		description = "Highlights the following words in chat, separated by commas",
```